src/main/java/org/storageart/storageart/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package org.storageart.storageart.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.storageart.storageart.service.UserService;
+import org.springframework.security.crypto.password.NoOpPasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Autowired
+    private UserService userService;
+
+    @Override
+    public void configure(HttpSecurity http) throws Exception{
+
+        http
+                .authorizeHttpRequests((requests) -> requests
+                        .antMatchers("/", "/home").permitAll()
+                        .anyRequest().authenticated()
+                )
+                .formLogin((form) -> form
+                        .loginPage("/login")
```

Comment:
You are using spring's login page, right?

Reply:
Yeah